### PR TITLE
Fix "hf mf chk" <t|d|s|ss> flags.

### PR DESCRIPTION
**Pull request by hsanjuan**
The command flags are broken.

"dss" or "tss" just fail.

"ds" ignores "d"

"ts" ignores "t"

This should fix it.

Note probably other commands (nested) are broken too.

### CODE REVIEW

**Comment by RfidResearchGroup** [comment]
doing this fix for RRG repo aswell?   Have you tried yubi stuff on the FIDO commands merlook pushed?

File: client/cmdhfmf.c [hunk 1]
```diff
@@ -1088,32 +1089,34 @@ int CmdHF14AMfChk(const char *Cmd)
...
-			timeout14a = 5000; // very slow
-		}
-		if (param3InUse && clen == 3 && (ctmp3[2] == 's' || ctmp3[2] == 'S')) {
-			timeout14a = 5000; // very slow
+	param_getstr(Cmd, 2, ctmp3, sizeof(ctmp3));
+
+	for (i = 0; i < sizeof(ctmp3); i++) {
+		ctmp = ctmp3[i];
+		if (!createDumpFile && (ctmp == 't' || ctmp == 'T')) {
+			transferToEml = 1; // mutually exclusive with d
```
[the remark this answers is not in the document]
Comment (hsanjuan) [reply client/cmdhfmf.c]: Because that's how it was before

**Comment by hsanjuan** [comment]
Sorry, I was naïve and assumed there was a single codebase to contribute to when I wrote this. This problem does not exist on iceman/rrg forks. Since it's horrible to contribute anything to 3 different repos which should just be branches of the same thing and the whole thing is a mess that doesn't even allow to cherry-pick things from one place to the other, I'd might as well just spend my limited time more effectively somewhere else. I'll keep an eye and maybe come back if this improves. ¯\_(ツ)_/¯

File: client/cmdhfmf.c [hunk 2]
```diff
@@ -1040,7 +1040,7 @@ int CmdHF14AMfChk(const char *Cmd)
 	int	keycnt = 0;
 	char ctmp	= 0x00;
 	int clen = 0;
-	char ctmp3[3]	= {0x00};
+	char ctmp3[4]	= {0x00,0x00,0x00,0x00};
```
Comment (merlokk) [inline client/cmdhfmf.c]: here can be just one zero value

**Comment by pwpiwi** [comment]
This has been withdrawn. See #718 instead.

**Pull request closed without merging**